content/php/concepts/math-functions/terms/deg2rad/deg2rad.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'deg2rad()'
+Description: 'Converts the number in degrees to the radian equivalent.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+  - 'Math Functions'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`deg2rad()`** function converts the number in degrees to the radian equivalent.
+
+## Syntax
+
+```pseudo
+$result = deg2rad(n);
+```
+
+Where `n` is the number in degrees to be converted to the radian equivalent.
+
+The result returns a value type of `float`.
+
+## Example
+
+```php
+echo deg2rad(90);
+// Output: 1.5707963267949
+```
+
+The result will be a float value of `1.5707963267949` as the function converts degree `90` to the radian equivalent.
+
+## Codebyte Example
```

Comment:
I would recommend deleting the codebyte because it's not really adding any information to this entry. Ideally every example is illustrating some specific functionality that isn't demonstrated in the others. Generally, we shouldn't have examples that are duplicative. Sometimes in entries that cover many elements, such as an entry on conditionals, we may have a summary section that has examples similar to those presented earlier.